### PR TITLE
Capsule update docs update

### PR DESCRIPTION
**Pull request by JanPrusinowski**

### CODE REVIEW

File: docs/variants/protectli_fw6/building-manual.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Building manual
```
Comment (krystian-hebel) [inline docs/variants/protectli_fw6/building-manual.md:1]: Build instructions for Protectli were unified in https://github.com/Dasharo/docs/pull/851. Don't re-add this file, modify https://github.com/Dasharo/docs/blob/master/docs/unified/protectli/building-manual.md instead.

File: docs/variants/protectli_v1x10/building-manual.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
+# Building manual
```
Comment (krystian-hebel) [inline docs/variants/protectli_v1x10/building-manual.md:1]: As above.

File: docs/variants/protectli_vp2410/building-manual.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
+# Building manual
```
Comment (krystian-hebel) [inline docs/variants/protectli_vp2410/building-manual.md:1]: As above.

File: docs/variants/protectli_vp2420/building-manual.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
+# Building manual
```
Comment (krystian-hebel) [inline docs/variants/protectli_vp2420/building-manual.md:1]: As above.

File: docs/variants/protectli_vp46xx/building-manual.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
+# Building manual
```
Comment (krystian-hebel) [inline docs/variants/protectli_vp46xx/building-manual.md:1]: As above.

File: docs/variants/protectli_vp66xx/building-manual.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
+# Building manual
```
Comment (krystian-hebel) [inline docs/variants/protectli_vp66xx/building-manual.md:1]: As above.

**Review by krystian-hebel — CHANGES_REQUESTED** [review]

File: docs/variants/protectli_fw6/building-manual.md (hunk 1 above)
Comment (JanPrusinowski) [reply docs/variants/protectli_fw6/building-manual.md]: Changed in: d0e64d58a768bd1ac17b02f724b3524074553fa4

File: docs/variants/protectli_v1x10/building-manual.md (hunk 2 above)
Comment (JanPrusinowski) [reply docs/variants/protectli_v1x10/building-manual.md]: Changed in: d0e64d58a768bd1ac17b02f724b3524074553fa4

File: docs/variants/protectli_vp2410/building-manual.md (hunk 3 above)
Comment (JanPrusinowski) [reply docs/variants/protectli_vp2410/building-manual.md]: Changed in: d0e64d58a768bd1ac17b02f724b3524074553fa4

File: docs/variants/protectli_vp2420/building-manual.md (hunk 4 above)
Comment (JanPrusinowski) [reply docs/variants/protectli_vp2420/building-manual.md]: Changed in: d0e64d58a768bd1ac17b02f724b3524074553fa4

File: docs/variants/protectli_vp46xx/building-manual.md (hunk 5 above)
Comment (JanPrusinowski) [reply docs/variants/protectli_vp46xx/building-manual.md]: Changed in: d0e64d58a768bd1ac17b02f724b3524074553fa4

File: docs/variants/protectli_vp66xx/building-manual.md (hunk 6 above)
Comment (JanPrusinowski) [reply docs/variants/protectli_vp66xx/building-manual.md]: Changed in: d0e64d58a768bd1ac17b02f724b3524074553fa4

**Review by krystian-hebel — APPROVED** [review]

**Pull request merged**